renderer-macos/lib/bindings/BriskMenu.re

```diff
@@ -1,7 +1,8 @@
 open CocoaTypes;
 
+type t = menu;
+
 type kind =
-  | Apple
```

Comment:
Why did you remove `Apple`?

Reply:
Because apple is an unsupported legacy API.

Reply:
Gotcha. 🤦‍♂️